### PR TITLE
[UI] [Chore] Add error for unwanted console.logs in the UI

### PR DESCRIPTION
**Pull request by dianabirs**
- added rule to mark console logs as errors by linter
- added exceptions in the files that have necessary log statements
- left a `console.log` in `ui/apps/everest/src/components/nav-item/NavItem.tsx` for visibility (will remove before merging)

### CODE REVIEW

File: .github/workflows/dev-fe-gatekeeper.yaml [hunk 1]
```diff
@@ -2,18 +2,8 @@
...
-      - ".github/workflows/dev-fe-e2e.yaml"
     branches:
-      - main
-  pull_request:
-    paths:
-      - "ui/**"
-      - ".github/workflows/dev-fe-gatekeeper.yaml"
-      - ".github/workflows/dev-fe-e2e.yaml"
-    types: [opened, synchronize]
+      - chore/add-eslint-warning-for-logs
```
Comment (oksana-grishchenko) [inline .github/workflows/dev-fe-gatekeeper.yaml:6]: looks like these changes should be reverted, we probably still want the workflow to be triggered by old triggers, not only by pushing to the current branch.

**Review by oksana-grishchenko — APPROVED** [review]

File: ui/apps/everest/src/pages/settings/default-configurations/default-configurations.tsx [hunk 2]
```diff
@@ -56,6 +56,7 @@ export const DefaultConfigurations = () => {
   });
 
   const onSubmit: SubmitHandler<DefaultConfigurationsType> = (data) => {
+    // eslint-disable-next-line no-console
     console.log(data);
```
Comment (fabio-silva) [inline ui/apps/everest/src/pages/settings/default-configurations/default-configurations.tsx:60]: do we need this log? I think we can just remove it

File: ui/apps/everest/src/components/nav-item/NavItem.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ export const NavItem = ({
   onClick,
   ...listItemProps
 }: NavItemProps) => {
+  console.log('🚀 ~ open:', open);
```
Comment (fabio-silva) [inline ui/apps/everest/src/components/nav-item/NavItem.tsx:22]: should we then add the ignore comment here?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: ui/apps/everest/src/components/nav-item/NavItem.tsx (hunk 3 above)
Comment (fabio-silva) [reply ui/apps/everest/src/components/nav-item/NavItem.tsx:22]: never mind, it's gonna be removed, gotcha

Comment (dianabirs) [reply ui/apps/everest/src/components/nav-item/NavItem.tsx:22]: I left this log intentionally so that the pipeline failure would be visible

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**